### DISCUSSION TITLE
Proposal <podcast:borrow>

### DISCUSSION THREAD

**Discussion by PlayBoxTech** [Podcastindex-org/podcast-namespace#494] [Spec Proposal]
For several years in the Podcast realm, its been common to have an episode of Podcast A include all of an episode of Podcast B in their feed, usually with an introduction. Which is usually intended to get you to subscribe to Podcast B, if you aren't already. However, if you are already subscribed to both Podcast A and B, you wind up getting a double download of essentially the same episode. 

```<podcast:borrow><url of episode></podcast:borrow>```

**Comment by theDanielJLewis** [comment]
This is what we've been trying to figure out with `<podcast:remoteItem>` (#481).
